spec/system/task_spec.rb

```diff
@@ -84,9 +77,9 @@
       end
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
-        # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        # TODO: FactoryBotのtraitを利用してください ok
+        project = create(:project)
+        task = create(:task, :status_done, project_id: project.id)
```

Comment:
associationを設定したことで`project_id`の指定が不要になるかと思うので、ご確認ください。
```suggestion
        task = create(:task, :status_done)
```

Reply:
99a0a01
7468dc8
にて修正しました。